### PR TITLE
Improved argparse

### PR DESCRIPTION
**Pull request by pennyfx**
1. `datmo` shows help message now
2. Each module can show its own usage notes by implementing an def usage(self): function
3. Started migrating argparse messages to en.py

### CODE REVIEW

File: datmo/cli/parser.py [hunk 1]
```diff
@@ -1,223 +1,219 @@
...
-    default=None,
-    nargs="?",
-    type=str,
-    help="pass in the session id to list the tasks in that session")
-
-# Task stop arguments
-task_stop = task_subcommand_parsers.add_parser("stop", help="stop tasks")
-task_stop.add_argument(
-    "--id", dest="id", default=None, type=str, help="task id to stop")
+
```
Comment (asampat3090) [inline datmo/cli/parser.py]: I'll have to test this with the docs code to see if sphinx is able to pick it up. not sure if it will

Comment (asampat3090) [reply datmo/cli/parser.py]: Not sure how this works but meant to reference the entire file and specifically the get_datmo_parser function

Comment (pennyfx) [reply datmo/cli/parser.py]: Looks like this should work

https://sphinx-argparse.readthedocs.io/en/stable/usage.html

```
.. argparse::
   :module: my.module
   :func: my_func_that_returns_a_parser
   :prog: fancytool
```

**Comment by pennyfx** [comment]
Fixes #94

**Comment by asampat3090** [comment]
had to add some error handing for sessions to match task and snapshot. otherwise this is good for merge. branch on this repo is at PR #110

**Pull request closed without merging**